src/components/ButtonDropdown/ButtonDropdown.tsx

```diff
@@ -180,9 +171,7 @@ export const ButtonDropdown = ({
       {dropdownMenuTriggerWithProps}
       <DropdownMenu
         className={styles['button-dropdown__dropdown-menu']}
-        handleOnClick={closePanel}
-        handleOnEscDown={(e) => handleKeyDown(e)}
-        handleOnTabDown={(e) => handleKeyDown(e)}
```

Comment:
Since not handling key presses here, just have to send close dropdown menu callback